src/menu.ts

```diff
@@ -0,0 +1,55 @@
+import { Menu, shell } from 'electron'
+import { getPlatform } from './utils/electron'
+
+const isDevEnv = process.env.NODE_ENV === 'development'
+const isMac = getPlatform() === 'mac'
+
+// Custom application menu
+// https://www.electronjs.org/docs/latest/api/menu
+const template: Electron.MenuItemConstructorOptions[] = [
+  ...getAppMenu(),
+  { role: 'fileMenu' },
+  { role: 'editMenu' },
+  {
+    // { role: 'viewMenu' }
+    label: 'View',
+    submenu: [
+      ...getDevToolsMenu(),
+      { type: 'separator' },
+      { role: 'resetZoom' },
+      { role: 'zoomIn' },
+      { role: 'zoomOut' },
+      { type: 'separator' },
+      { role: 'togglefullscreen' },
+    ],
+  },
+  { role: 'windowMenu' },
+  {
+    role: 'help',
+    submenu: [
+      {
+        label: 'Report an issue',
+        click: async () => {
+          await shell.openExternal(
+            'https://github.com/grafana/k6-studio/issues'
+          )
+        },
+      },
+    ],
+  },
+]
+
+function getAppMenu(): Electron.MenuItemConstructorOptions[] {
+  return isMac ? [{ role: 'appMenu' }] : []
+}
+
+function getDevToolsMenu(): Electron.MenuItemConstructorOptions[] {
+  return isDevEnv
+    ? [{ role: 'reload' }, { role: 'forceReload' }, { role: 'toggleDevTools' }]
```

Comment:
Shortcuts are bound with the application menu. Here, we can conditionally remove them from prod mode.